tests/conftest.py

```diff
@@ -88,22 +88,31 @@ def initialize_environment(request, environ):
 
 
 @pytest.fixture
-def context():
-    client = mock.Mock(
-        project="testing",
-        namespace=None,
-        spec=("project", "namespace"),
-        stub=mock.Mock(spec=()),
-    )
-    context = context_module.Context(
-        client,
-        eventloop=TestingEventLoop(),
-        datastore_policy=True,
-        legacy_data=False,
-    )
+def context_factory():
+    def context(**kwargs):
+        client = mock.Mock(
+            project="testing",
+            namespace=None,
+            spec=("project", "namespace"),
+            stub=mock.Mock(spec=()),
+        )
+        context = context_module.Context(
+            client,
+            eventloop=TestingEventLoop(),
+            datastore_policy=True,
+            legacy_data=False,
+            **kwargs,
```

Comment:
Python 2.7 doesn't like the trailing comma here.

Reply:
5cd8e70